pages/profiles/graph.svg.js

```diff
@@ -0,0 +1,29 @@
+// node_modules
+import Image from "next/image";
+
+// components
+import Breadcrumbs from "../../components/breadcrumbs";
+import PagePreamble from "../../components/page-preamble";
+import { DataPanel, DataAreaTitle } from "../../components/data-area";
+
+export default function GraphSvg() {
+  return (
+    <>
+      <Breadcrumbs />
+      <PagePreamble />
+      <>
+        <DataAreaTitle>Graph</DataAreaTitle>
+        <DataPanel>
+          <a href="https://api.data.igvf.org/profiles/graph.svg">
```

Comment:
This always goes to the production server to retrieve the svg, but it’d be nice to go to the server attached to whatever UI instance this runs on. You can import `API_URL` from lib/constants.js and compose the URL from that. `API_URL` for local instances holds `http://localhost:3000`.